sum-of-polygon-angles.js

```diff
@@ -1,8 +1,3 @@
-/**
- * @param {number} sides - number of sides in a polygon
- * @return {number} - returns sum of the internal angles of a polygon
- */
-
 function sumOfInternalAngles(sides) {

```

Comment:
/**
 * Returns sum of the internal angles of a polygon
 * @param {number} sides - number of sides in a polygon
 * @returns {number} - returns sum of internal angles of a polygon
 */